app/client/src/entities/Action/index.ts

```
@@ -82,6 +82,12 @@ export interface BodyFormData {
   type: string;
 }
 
+export interface AutoGeneratedHeader {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AutoGeneratedHeader'. Consider to delete it or rename it.

'AutoGeneratedHeader' is defined here:


(lint/suspicious/noRedeclare)